apps/web/src/services/asset-inventory/components/CollectorHistoryJobTaskTable.vue

```diff
@@ -160,7 +158,7 @@ const getQuery = () => {
 
 /* Components */
 const handleSelect = (selectedIndexes: number) => {
-    emit('select', state.items[selectedIndexes[0]]);
+    emit('select', items.value[selectedIndexes[0]]);
```

Comment:
nitpick:
```suggestion
    emit('select', items.value?.[selectedIndexes[0]]);
```

or early return or add warning!
```
    if (!items.value || !items.value?.length) return; // add console.warn(...)
    emit('select', items.value[selectedIndexes[0]]);
```

Reply:
thanks!